home/views.py

```diff
@@ -7,3 +9,10 @@ def about(request):
 
 def landingpage(request):
     return render(request, 'home/landingpage.html')
+
+
+class QuestionsListView(FilterView):
```

Comment:
Not sure how can you wrote tests to the filter view, probably by using a client that posts various filter selections and verifies the results, but you need to have tests that verify that all the filtering options you intended to provide are indeed being provided.

Reply:
> Not sure how can you wrote tests to the filter view, probably by using a client that posts various filter selections and verifies the results, but you need to have tests that verify that all the filtering options you intended to provide are indeed being provided.

@ifireball 
I wrote tests for the filter view in PR #83 (In this PR there is no option to filter questions yet because it contains only the base of the explore page ) .

Reply:
If the filter view isn't doing anything at this point, why introduce it to the code? Might as well make the code simpler for now and only add it when you need it and you can test it.

Reply:
@ifireball 
Ok I convert it into ListView instead of Filter View